src/calculateRentalCost.js

```diff
@@ -21,6 +21,15 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const rentCosts = 40;
+  const totalRent = rentCosts * days;
+  if (days >= 7) {
+    return totalRent - 50;
+  } else if (days >= 3) {
+    return totalRent - 20;
+  } else {
```

Comment:
no need for this else
```suggestion
  } 
```